arm-sql/2014-04-01/swagger/serverCommunicationLinks.json

```diff
@@ -0,0 +1,308 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database",
+    "description": "Provides create, read, update and delete functionality Azure SQL server communication links.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/communicationLinks/{communicationLinkName}": {
+      "delete": {
+        "tags": [
+          "ServerCommunicationLinks"
+        ],
+        "operationId": "CommunicationLinks_Delete",
```

Comment:
Is there a reason for this documentation, per all other operationIds this should be `CommunicationLinks_delete`